salt/cli/run.py

```diff
@@ -31,7 +31,7 @@ def run(self):
             if check_user(self.config["user"]):
                 pr = salt.utils.profile.activate_profile(profiling_enabled)
                 try:
-                    ret = runner.run()
+                    ret = runner.run(full_return=True)
```

Comment:
We're making code changes here. Does this warrant a changelog, or would it be considered alongside your original PR?

Also changing `full_return` for this file and salt/runner.py will this change behavior for users?